CalibTracker/SiStripChannelGain/test/7TeVData/certif.json

```diff
@@ -0,0 +1 @@
l": "GOOD"}, "297486": {"comment": ", strip: SiStrip: 98.6%; TID/2-: 93.2%", "qual": "GOOD"}, "275326": {"comment": "SiStrip is  out of DAQ., strip: not in DAQ, pixel: Maps show effect of bad tracking perfromance", "qual": "BAD,"}, "279116": {"comment": "", "qual": "GOOD"}, "296870": {"comment": "as per whiteboard instruction, pixel: as per whiteboard instruction", "qual": "BAD,"}, "280363": {"comment": "", "qual": "GOOD"}, "279115": {"comment": "", "qual": "GOOD"}, "275000": {"comment": "", "qual": "GOOD"}, "297673": {"comment": ", strip: SiStrip: 99.1%", "qual": "GOOD"}, "297488": {"comment": "", "qual": "GOOD"}, "276810": {"comment": "", "qual": "GOOD"}, "297723": {"comment": "", "qual": "GOOD"}, "297722": {"comment": "", "qual": "GOOD"}, "276811": {"comment": "", "qual": "GOOD"}, "295610": {"comment": "Whiteboard instructions, strip: SiStrip: 99.2%, pixel: Whiteboard instructions", "qual": "BAD,"}, "286033": {"comment": "", "qual": "GOOD"}, "281693": {"comment": "", "qual": "GOOD"}, "281691": {"comment": "", "qual": "GOOD"}, "283548": {"comment": "", "qual": "GOOD"}, "296969": {"comment": ", strip: SiStrip:99.1%", "qual": "GOOD"}, "296801": {"comment": ", pixel: cluster-on-track plot difficult to judge. o/wise good.", "qual": "GOOD"}, "296800": {"comment": "as per whiteboard instruction , pixel: as per whiteboard instruction", "qual": "BAD,"}, "294939": {"comment": "Distributions are different from reference., strip: SiStrip:99.2%, pixel: No reference run", "qual": "BAD,"}, "296966": {"comment": ", strip: Sistrip:99.1%\n", "qual": "GOOD"}, "296967": {"comment": ", strip: Sistrip:99.1%\n", "qual": "GOOD"}, "283469": {"comment": "", "qual": "GOOD"}, "278976": {"comment": "", "qual": "GOOD"}, "283049": {"comment": "very low stats, strip: very low stats.. &quot;HighStripNoise_Apvcycle&quot; plot is empty, pixel: very low stats.", "qual": "GOOD"}, "283407": {"comment": "", "qual": "GOOD"}, "274146": {"comment": "All looks fine, strip: All looks fine. TID -2 efficiency value is a known issue and within expected range., pixel: All looks fine", "qual": "GOOD"}, "276456": {"comment": "all Strip partition are NOT_READY (HV_OFF), strip: not in DAQ, pixel: Pixel OFF most the time, histograms empty", "qual": "BAD "}, "295342": {"comment": "Following Whiteboard instructions, strip: SiStrip: 99.3%, pixel: Following Whiteboard instructions", "qual": "BAD,"}, "274142": {"comment": "HV and GT vs lumi is false for entirety of the run., strip: HV and GT vs lumi is false for entirety of the run., pixel: HV and GT vs lumi is false for entirety of the run.", "qual": "BAD "}, "299369": {"comment": ", strip: SiStrip:99.1%", "qual": "GOOD"}, "299368": {"comment": ", strip: SiStrip:99.2%", "qual": "GOOD"}, "299450": {"comment": ", strip: SiStrip: 99.2%;", "qual": "GOOD"}, "296089": {"comment": "Whiteboard instructions, pixel: Whiteboard instructions", "qual": "BAD,"}, "296980": {"comment": ", strip: Sistrip:99.1% TID/2-94.96%\tTID/2+:93.23%", "qual": "GOOD"}, "296668":
```

Comment:
Dear @davidlange6, 
You are right, this information can (and is during the production) derived from other tools and I shouldn't have added in the pull-request in the first place.
I will correct it.
Thanks for pointing this out.